modules/user-access/pages/jwt-token.adoc

```diff
@@ -235,6 +235,11 @@ Both instances, users need to specify the URL of the public key or secret in the
 === Non-Interactive
 Users can configure JWT token non-interactively by using the `gadmin` CLI tool by running one of these commands for RSA or HMAC.
 
+[NOTE]
+====
+The JWT token generated in this section is only supported by restpp side.
```

Comment:
refactor to:
The JWT token issued by a third party is limited to authentication on the RESTPP side exclusively. To enable authentication for both GSQL and RESTPP services, it is recommended to generate the token using the GSQL API service.